app/src/routes/family/drive/$driveId/form/consent.tsx

```
@@ -0,0 +1,107 @@
+        onSubmit={(e) => {
+          e.preventDefault();
+          e.stopPropagation();
+          form.handleSubmit();
+        }}
+      >
+        <CardContent>
+          <FieldGroup>
+            {/* First checkbox - Address sharing consent */}
+            <form.Field
+              name="consentGiven"
+              children={(field) => (
```

Biome (2.4.4)

[error] Avoid passing children using a prop

(lint/correctness/noChildrenProp)